lib/module.php

```diff
@@ -629,3 +635,90 @@ function solusiovps_restart(array $params)
         return $e->getMessage();
     }
 }
+
+function solusiovps_ListAccounts(array $params)
+{
+    try {
+        $statusMap = [
+            "0" => WHMCS\Service\Status::ACTIVE,
+            "1" => WHMCS\Service\Status::SUSPENDED,
+        ];
+        $accounts = [];
+
+        $serverId = (int)$params['serverid'];
+        $serverParams = Server::getParams($serverId);
```

Comment:
You can just put this variable in place, it used just once, no reason to extract it to an additional variable

`$serverParams = Server::getParams((int)$params['serverid']);`